templates/partials/question.html

```diff
@@ -5,7 +5,12 @@
 {% import 'macros/helpers.html' as helpers %}
 
 {% set form = content.form %}
-{% set question_title= helpers.interviewer_note(question.interviewer_only, question.title) %}
+
+{% if question.interviewer_only %}
+  {% set question_title= helpers.interviewer_note(question.title) %}
+{% else %}
+  {% set question_title = question.title %}
+{% endif %}
```

Comment:
It would be good if this was formatted as per the question_description, question_instruction calls below. Additionally, you should only need to import the interviewer_note macro, rather than the helpers partial.

Reply:
Formatted, imports sorted, thanks!👍